pkgs/cupertinohttp/src/dart-sdk/include/dart_api.h

```diff
@@ -0,0 +1,3909 @@
+/*
```

Comment:
was this file copied from somewhere? Is the long term plan to keep an independent copy here?

Reply:
AFAIK, the current situation is that every ffi program that uses the Dart API from native code has to embed this. @dcharkes can let me know if I'm wrong.

Reply:
how will we handle updates? will we let independent copies drift?

How did you get this copy?